posthog/models/kafka_debug/sql.py

```diff
@@ -0,0 +1,82 @@
+from dataclasses import dataclass
+from posthog.clickhouse.kafka_engine import kafka_engine
+from posthog.clickhouse.table_engines import MergeTreeEngine
+from posthog.settings import CLICKHOUSE_CLUSTER, CLICKHOUSE_DATABASE
+
+
+@dataclass
+class KafkaDebugKafkaTable:
+    brokers: list[str]
+    topic: str
+    consumer_group: str = "debug"
+
+    @property
+    def table_name(self) -> str:
+        return f"kafka_{self.topic}_debug"
+
+    def get_create_table_sql(self) -> str:
+        return f"""
+      CREATE TABLE IF NOT EXISTS `{CLICKHOUSE_DATABASE}`.{self.table_name} ON CLUSTER '{CLICKHOUSE_CLUSTER}'
+      (
+        payload String
+      )
+      ENGINE={kafka_engine(kafka_host=",".join(self.brokers), topic=self.topic, group=self.consumer_group)}
+      SETTINGS input_format_values_interpret_expressions=0, kafka_skip_broken_messages = 100
```

Comment:
I don't think we should ever skip messages here since we are just grabbing the payload as a string (no JSON deserialization) I'll remove the `kafka_skip_broken_messages` setting